src/Actions/Products/ImportProducts.php

```diff
@@ -89,13 +91,10 @@ public function delete(?string $store = null): void
 
             $productSkus = collect();
 
-            $productQuery = $productModel::selectOnlyIndexable()
-                ->where($flat->qualifyColumn('type_id'), 'configurable')
-                ->orWhereNotIn($flat->qualifyColumn('entity_id'), $childIds->toArray())
-                ->withEventyGlobalScopes('statamic.product.scopes');
+            $productQuery = $productModel::selectAttributes(['sku', 'name']);
```

Comment:
You can keep the custom statamic one, but remove the others. Something like:
```suggestion
            $productQuery = $productModel::selectAttributes(['sku', 'name'])
                ->withoutGlobalScopes()
                ->withEventyGlobalScopes('statamic.product.scopes');
```